docroot/themes/humsci/humsci_basic/src/scss/_base.scss

```diff
@@ -1,8 +1,18 @@
 @charset "UTF-8";
 
-// Decanter's use of global variables won't allow us to import with @use
-// @use 'decanter/core/src/scss/components/brand-bar/index';
-@import 'decanter/core/src/scss/decanter-no-markup';
+@import
+  'decanter/core/src/scss/decanter-no-markup',
+  'decanter/core/src/scss/core/normalize',
+  'decanter/core/src/scss/core/base',
+  'decanter/core/src/scss/core/helpers',
+  'decanter/core/src/scss/elements',
+  'decanter/core/src/scss/core/flex-grid'
```

Comment:
all this does is generate classes that can be applied to html, I think all the relevant flex and grid mixins are imported with decanter-no-markup